WDL/runtime/task.py

```diff
@@ -24,7 +24,9 @@
 from typing import Tuple, List, Dict, Optional, Callable, Iterable, Set, Any
 from contextlib import ExitStack
 import docker
-from .. import Error, Type, Env, Value, StdLib, Tree, _util
+
+import WDL
```

Comment:
I think this import may do something unexpected (like import the pip-installed version of miniwdl into our development version at runtime) and shouldn't be needed if we have suitable `from .. import ...` below